management/Makefile

```diff
@@ -1,33 +1,16 @@
-
-# The kname of the context for the management cluster
-# The name of the context for your Kubeflow cluster
 NAME=$(shell yq r ./instance/settings.yaml name)
+PROJECT=$(shell yq r ./instance/settings.yaml project)
```

Comment:
Can we add an error message here if ./instance/settings.yaml is still empty/the default value?
similarly for others

Reply:
Since the setting values is one step also overriding values doesn't make any change with `kpt cfg set`. I have combined the values and made one make rule.

Output if values are not set.
```zsh
▶ make apply
either of <NAME>, <ZONE_OR_REGION>, <PROJECT_TO_DEPLOY_IN> not set.
make: *** [validate-values] Error 1
```